pkg/agent/utils.go

```diff
@@ -586,3 +586,25 @@ func addFeatureGateString(featureGates string, key string, value bool) string {
 	}
 	return strings.Join(pairs, ",")
 }
+
+// ParseCSE parses the raw VMSS CSE output
+func ParseCSE(message string) (datamodel.VMSSInstanceViewCSEStatus, error) {
```

Comment:
looks like we need to remove vmss from datamodel.VMSSInstanceViewCSEStatus too. This data structure applies for vmas too

Reply:
+1

Reply:
change it to InstanceViewCSEStatus